docker/docker.go

```diff
@@ -400,11 +400,11 @@ func StopContainer(ctx context.Context, dClient *client.Client, hostname string,
 	}
 	var err error
 	// define the stop timeout
-	stopTimeoutDuration := StopTimeout * time.Second
+	stopTimeoutDuration := StopTimeout
```

Comment:
don't need this anymore as `StopTimeout` can be used directly, same for `RestartTimeout` above.

Reply:
Yeah right. Updating it.